M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1299,6 +1455,14 @@ doc ///
         (Rn, ZZ, Ring)
         [Rn, Verbose]
         [Rn, ModP]
```

Comment:
The name "Rn" is too brief!  Think of something descriptive.

Reply:
RegularInCodimensionN?
That seemed rather long to type.  
I thought we wanted to avoid things like 
isRn
as a matter of style.  Although that would be another option.

I think Rn is the correct and extensively used mathematical name for this though.

Reply:
How about SerreRn, SerreSn?

Reply:
That's good!  Although I suppose isSerreSn is redundant.  (Since the S is Sn stands for Serre).  I don't think I have any particularly good ideas about how to implement Sn checking though.  There will be another improvement to this package when we get the randomRationalPoints working as well as we want, but that's probably for 1.17.

Reply:
"isSerreRn" is bad for two reasons: it contains "R", which is a symbol with no English meaning; it contains "n", which is just an a arbitrary identifier that could have been any other letter of the alphabet.

Reply:
Hi Dan, 
I guess we could just go with 
regularInCodimension
We presumably want to avoid abbreviations like
regInCodim
regularInCodim
On the other hand "Rn" is really the way this is used in mathematical speech I believe.  Specifically, people will really say things like this scheme/variety/ring is R0 / R1 / R2...
Let me know.

Reply:
I think `regularInCodimension` is a good (self-dcumenting) name.  The documentation could mention the name`Rn`.  I agree that even though it is standard math usage in commutative algebra, it is too short of a name, making it hard to see, search for, etc.  I guess also: the name of the function should then be `R`, not `Rn`, but that isn't allowed either.

Reply:
Ok, I've renamed it.